src/ontology/imports/foodon_product_import.ofn

```diff
@@ -92361,13 +92368,15 @@ AnnotationAssertion(rdfs:label obo:FOODON_03310790 "white bean (whole, dried)"@e
 EquivalentClasses(obo:FOODON_03310790 ObjectIntersectionOf(obo:FOODON_00002752 obo:FOODON_03301232))
 SubClassOf(obo:FOODON_03310790 obo:FOODON_03301232)
 
-# Class: obo:FOODON_03310791 (corn (on-the-cob, kernel or parts))
+# Class: obo:FOODON_03310791 (corn (whole or parts))
 
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03310791 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03310791 "A whole corn or some substance made from a processed corn."@en)
```

Comment:
Should be "A whole corn cob or some substance made from one or more processed corn cobs."

Reply:
I agree, thank you! I've just pushed the changes.